### PR TITLE
TeamsJS support matrix updates for 08_2023

### PR DESCRIPTION
**Pull request by mikerapid**
Updates
* Search Capability is now supported in Beta for Outlook Win 32 and for Outlook Owa 
* Reinstated Outlook on Web and Browser preview mark to pending 100% GA rollout to ensure universal customer access to GA experience.
*  Search is now supported in at least 1 host (Outlook), moved out of preview capabilities awaiting host support and added to Cross-host capabilities table.
* Added support for Mail on Outlook iOS/Android
* M365 app for iOS moved from preview to general availability

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c1a96718bfcf3cc3c1241c4c0df2cbce923716b1/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fed004c2-f45b-430b-9969-281954e67858) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Thanks for starting this @mobajemu-msft! A few asks:

- Could you undo the auto-formatting changes?
- Could you update the PR description with a list of exactly what capability/hub values you're changing?
- New update: please add preview support for: Mail on Outlook iOS/Android

**Comment by mobajemu-msft** [comment]
> Thanks for starting this @mobajemu-msft! A few asks:
> 
> * Could you undo the auto-formatting changes?
> * Could you update the PR description with a list of exactly what capability/hub values you're changing?
> * New update: please add preview support for: Mail on Outlook iOS/Android
> * New update: could you please add an asterisk (*) after each check in the **Outlook** for **web** and **desktop** columns across all capabilities? I prematurely removed them upon the start of GA support rollout, however I should have waited until they were 100% rollout, to guarantee all customers would get the GA experience. (You can just summarize this as a single list item in this PR description.) Thank you (and sorry for the extra work)!
> * New update: remove the asterisk for all **desktop** and **web** cells of M365 (aka Office, Harmony) app. It was just announced in shiproom that Harmony launch page support is 100% rolled out for GA

There was no asterisk to remove for **desktop** and **web** cells of M365

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/088f299e234465a6ece5a710df2183448e33197a/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a1ecbe50-80af-403c-ac8c-3056e6143085) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Search:

- Now that Search is supported in at least 1 host (Outlook), we need to move it out of [Preview capabilities awaiting host support](https://review.learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/teamsjs-support-m365?branch=pr-en-us-9156#preview-capabilities-awaiting-host-support) and add a row for it in [Cross-host capabilities](https://review.learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/teamsjs-support-m365?branch=pr-en-us-9156#cross-host-capabilities) table.

Mail:

- Don't forgot to add a check+askterisk for iOS/Android to the top-level [Cross-host capabilities](https://review.learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/teamsjs-support-m365?branch=pr-en-us-9156#cross-host-capabilities) table as well. Could you also double-check any other updates you made to make sure you captured them in both the top-level capabilities summary table(s) and also the individual capability section tables?
- Note this change as a bullet in the PR description

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e2ddcca8597e564874ba8ed8a2ee6625ab925b29/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2fabbc20-7ef3-41e6-b325-da124f09a58d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3d76e6d837c34ba0c365061b77a3165590763c55/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce67cee2-3542-4cfb-80d2-ace43ed44c65) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Oops, a few more changes:

**PR Description** (add bullets for these changes)
- Added support for Mail on Outlook iOS/Android
- M365 app for iOS moved from preview to general availability

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 1]
```diff
@@ -229,6 +229,24 @@ The following table lists host application support for TeamsJS capabilities that
             <td></td>
             <td></td>
         </tr>
+        <tr>
+            <th>search <img src="./images/preview-badge.png" alt="Badge indicating this capability is in preview" /></th>
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:233]: The text "search" should link to the search section of this doc, similar to other rows

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 2]
```diff
@@ -80,17 +80,17 @@ The following table lists host application support for TeamsJS capabilities that
             <th><a href="#app">app</a></th>
             <td>&#x2713;</td>
             <td>&#x2713;</td>
-            <td>&#x2713;</td>
+            <td>&#x2713*;</td>
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:83]: asterisk should go after semicolon (rendering as a typo)

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 3]
```diff
@@ -229,6 +229,24 @@ The following table lists host application support for TeamsJS capabilities that
...
+        <tr>
+            <th>search <img src="./images/preview-badge.png" alt="Badge indicating this capability is in preview" /></th>
+            <td></td>
+            <td></td>
+            <td>&#x2713;*</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>&#x2713;*</td>
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:241]: Oops, wrong Outlook column (this is for "new Outlook", with the PRE on the logo); should be marked for classic win32 Outlook in the column right before. (Same thing for table in "search" section of doc)

File: msteams-platform/m365-apps/teamsjs-support-m365.md (hunk 2 above)
Comment (mobajemu-msft) [reply msteams-platform/m365-apps/teamsjs-support-m365.md]: good catch thanks

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a5dd7f65a1e741c44bc40ad41581fa1b0e6d7bd/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a8a05df-f844-488e-8100-ec0cb9ba1270) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — APPROVED** [review]
Looks good!

**Comment by erikadoyle** [comment]
@surbhigupta12 this is ready from our side. Could someone help review/publish? Thank you!

**Comment by erikadoyle** [comment]
@surbhigupta12 gentle reminder for this--we're hoping to publish soon to fix a bug that Teams/MetaOS is tracking.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/126298548487056c58c32e6b4d03f6a42643c82e/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89c6ee36-da18-4b53-88d1-35315438b0c2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**